app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+
+### Get an international relocation payment (IRP) worth £10,000
+
+The IRP is designed to help pay the relocation expenses of languages or physics teacher trainees moving to the UK. 
+
+If you are eligible, you’ll receive a cash sum of £10,000 from DfE. You will not have to pay this money back and it can be spent in any way you wish. You can only receive the IRP once.
+
+
+#### Check your eligibility for the IRP
+
+To be eligible, teacher trainees must meet all 3 of the following requirements.
+
+You must have taken up a place on a teacher training course in England leading to [qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts)  for the academic year 2023 to 2024. Your teacher training provider must be accredited by the UK government and you must be training to teach one of the following:
+
+* any secondary language or languages except English, including courses which are made up of at least 50% languages 
+
+* secondary physics, including:
+
+  - engineers teach physics 
+  - courses which are made up of at least 50% physics 
+   
+You must come to the UK on one of the following visas:  
+
+* Skilled worker visa
+* Student visa 
+* Youth Mobility Scheme 
+* Family visa
+* UK Ancestry visa 
+* British National (Overseas) visa
+* High Potential Individual visa 
+* Afghan citizens resettlement scheme
+* Afghan Relocations and Assistance Policy
+* Ukraine Family Scheme visa
+* Ukraine Sponsorship Scheme
+
+You must move to the UK no more than 3 months before the start of your course.  
+
+You will not be eligible for the IRP if you are:
+
+* already living in the UK 
+* undergoing [assessment only QTS](/assessment-only-providers) 
+
+[Learn more about applying for a visa to train to teach in England](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa).
+
+#### How the IRP will be paid
+
+If you have a place on a teacher training course where you pay a fee, you do not need to apply for the IRP. Your training provider will email you if your choice of course makes you eligible for the IRP. Your visa status and date of arrival will be checked by DfE. If you meet the eligibility requirements, you’ll get the IRP around the end of your first term.
+
+If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to express your interest in the IRP. We’ll be in touch when IRP applications open for salaried trainees in autumn 2023. If your application is successful, you’ll get the IRP around the end of your first term.
+
+You will not need to pay an agent or any other third party to complete your application. If you have had a message asking you to apply for or contact someone about the IRP, [this could be fraudulent](https://www.gov.uk/report-suspicious-emails-websites-phishing).
```

Comment:
I'm not sure this link text really works - it doesn't tell the user where the link will take them. And I'm not sure if this link is necessary?

Reply:
Agree - this link text does not meet accessibility standards so needs to be changed (or removed entirely)